docs/index.md

```diff
@@ -6,6 +6,32 @@ koishi-plugin-puppeteer 提供了通用的浏览器服务，可用于网页截
 为了正常使用这个插件，你首先需要确保你的电脑上已经安装有 Chrome / Edge / Firefox。同时，我们建议你保持浏览器和本插件的更新，因为版本不匹配可能会导致本插件无法正常使用。
 :::
 
+::: warning
+Ubuntu 在安装 Chrome 时可能会使用 snap 进行隔离。这将导致无法访问大部分主机文件夹或挂载点，并且 Koishi 可能会输出如下警告：
+
+```bash
+[W] app Error: net::ERR_FILE_NOT_FOUND at file:///your-koishi-workspace-path/node_modules/koishi-plugin-puppeteer/index.html
+```
+
+可以进行如下方法解决：
+- 使用传统 `deb` 包进行安装
+    ```bash
+    > curl -o https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb chrome.deb
+    ...
+    > dpkg chrome.deb
+    ...
```

Comment:
What is these symbols? `...`?

Reply:
Ignored output, maybe I should delete it